README.rst

```diff
@@ -59,6 +59,10 @@ See `sample instructions <https://bennierobinson.com/programming/2016/01/24/perl
 
 Install `MiKTeX <https://miktex.org/download>`_
 
+Alternatively, you can `install WSL <https://learn.microsoft.com/en-us/windows/wsl/install>`_
```

Comment:
Is there a stray space at the end here?

Reply:
It's a trailing underscore which turns the text "install WSL" into a hyperlink with that embedded URL. The docs say to use a double underscore there though, for an anoymous reference: 

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links

But a single underscore works in this case and is consistent with the link above.